redaxo/src/core/lib/config_db.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @psalm-readonly
+ */
+class rex_config_db
+{
+    /**
+     * @var string
+     */
+    public $host;
+    /**
+     * @var string
+     */
+    public $login;
+    /**
+     * @var string
+     */
+    public $password;
+    /**
+     * @var string
+     */
+    public $name;
+    /**
+     * @var bool
+     */
+    public $persistent;
+
+    /**
+     * @var string|null
+     */
+    public $ssl_key;
```

Comment:
Ich versuche einheitlich camelCase für Properties zu nutzen in Neucode.

Reply:
Ich hatte 1:1 die keys aus der config.yml genommen. Kann es aber gerne auf camelcase ändern

Reply:
Nicht total wichtig, aber ich fände es besser in camelCase.

Reply:
Machen wir doch gerne ☺️